homeassistant/components/arcam_fmj/media_player.py

```diff
@@ -137,13 +157,18 @@ def _stopped(host: str) -> None:
     async def async_update(self) -> None:
         """Force update of state."""
         _LOGGER.debug("Update state %s", self.name)
-        await self._state.update()
+        try:
+            await self._state.update()
+        except ConnectionFailed as connection:
+            _LOGGER.debug("Connection lost during update: %s", connection)
```

Comment:
Must swallow the exception here since we don't catch these cleanly without backtrace at the moment.